Userland/Libraries/LibGfx/ImageFormats/WebPLoader.cpp

```diff
@@ -755,7 +757,7 @@ Transform::~Transform() = default;
 class PredictorTransform : public Transform {
 public:
     static ErrorOr<NonnullOwnPtr<PredictorTransform>> read(WebPLoadingContext&, LittleEndianInputBitStream&, IntSize const& image_size);
-    virtual ErrorOr<void> transform(Bitmap&) override;
+    virtual ErrorOr<NonnullRefPtr<Bitmap>> transform(NonnullRefPtr<Bitmap>);
```

Comment:
```suggestion
    virtual ErrorOr<NonnullRefPtr<Bitmap>> transform(NonnullRefPtr<Bitmap>) override;
```

(should fix CI build error, too)